client/Components/NearbyTransitList.js

```diff
@@ -8,15 +8,19 @@ const List = styled.ul`
   padding: 0;
   `
 
-const NearbyTransitList = ({transitOptions}) => (
-  <>
-    <h3>NEARBY TRANSIT</h3>
-    <List>
-    { transitOptions.map((option, i) => (
-        <NearbyTransit key={i} name={option} />
-      )) }
-    </List>
-  </>
-);
+const NearbyTransitList = ({nearbyTransits}) => {
+
+  return (
+    <>
+      <h3>NEARBY TRANSIT</h3>
+      <List>
+      { nearbyTransits.map((option, i) => (
+          <NearbyTransit key={i} option={option} />
```

Comment:
If you have a true id, such as from the API response, (that is better than index)[https://reactjs.org/docs/lists-and-keys.html]. That way the ID isn't relative to the array order, and React can stay effecient.

Reply:
updated